backend/app/notify/telegram.go

```diff
@@ -171,6 +171,19 @@ func buildMessage(req Request) ([]byte, error) {
 // returns HTML with only tags allowed in Telegram HTML message payload, also trims ending newlines
 // https://core.telegram.org/bots/api#html-style
 func telegramSupportedHTML(htmlText string) string {
+	// telegram not allow h1-h6 tags
```

Comment:
Thanks for your contribution! That way of working with HTML would not work in the wild but will work here well as we process HTML which is produced by us previously, so it doesn't have anything unusual like extra spaces (`<h1 >`) or attributes (`<h1 class="red">`).

Would you be so kind as to move that code to a separate function, like `adjustHtmlTags`? Also, I would propose renaming `htmlText` to something like `adjustedHtmlText` so it will be obvious to the reader that the return statement operates with another string that was passed into that function.